Peer Dependency conflict between cli-build and cli

## 🐛 Bug report

A dependency error from @nebula.js/cli@3.0.0-alpha1 with peer @nebula.js/cli-build@"^2.x" is preventing proper creation of projects.

### Steps to Reproduce

1. install nebula.js:         npm install @nebula.js/cli @nebula.js/cli-build @nebula.js/cli-create @nebula.js/cli-serve --location=global
2. run nebula create:     nebula create bug

### Expected behavior

Installation of nebula, and proper creation of an extension

### Actual behavior

Continually run into a dependency error. Have uninstalled and reinstalled nebula on system to check for install issues.

### Versions

- nebula.js: 3.0.0-alpha1
- Platform: Win 11
- CLI: Git Bash

### Output

```
MINGW64 ~/Documents/Code/qlik/bug
$ nebula create bug
? Pick a picasso template (Use arrow keys)
? Pick a picasso template none


> Generating files...
> Installing dependencies...


npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: bug@0.1.0
npm ERR! Found: @nebula.js/cli-build@3.0.0-alpha1
npm ERR! node_modules/@nebula.js/cli-build
npm ERR!   dev @nebula.js/cli-build@"3.0.0-alpha1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @nebula.js/cli-build@"^2.x" from @nebula.js/cli@3.0.0-alpha1
npm ERR! node_modules/@nebula.js/cli
npm ERR!   dev @nebula.js/cli@"3.0.0-alpha1" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\ppolk\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\ppolk\AppData\Local\npm-cache\_logs\2022-06-23T14_24_45_806Z-debug-0.log


> Something went wrong when running npm install, try running the command yourself.
> Get started with the following commands:


   cd bug
   npm install
   npm run start
```

--- a/apis/stardust/types/index.d.ts
+++ b/apis/stardust/types/index.d.ts
@@ -400,16 +400,6 @@ declare namespace stardust {
         type: "dimension" | "measure";
     }
 
-    /**
-     * An object literal containing meta information about the plugin and a function containing the plugin implementation.
-     */
-    interface Plugin {
-        info: {
-            name: string;
-        };
-        fn: ()=>void;
-    }
-
     interface LoadType {
         (type: {
             name: string;
@@ -422,6 +412,16 @@ declare namespace stardust {
         version?: string;
         load: stardust.LoadType;
         meta?: object;
+    }
+
+    /**
+     * An object literal containing meta information about the plugin and a function containing the plugin implementation.
+     */
+    interface Plugin {
+        info: {
+            name: string;
+        };
+        fn: ()=>void;
     }
 
     interface ActionToolbarElement extends HTMLElement{

--- a/commands/cli/package.json
+++ b/commands/cli/package.json
@@ -22,9 +22,9 @@
     "lint": "eslint src"
   },
   "peerDependencies": {
-    "@nebula.js/cli-build": "^2.x",
-    "@nebula.js/cli-sense": "^2.x",
-    "@nebula.js/cli-serve": "^2.x"
+    "@nebula.js/cli-build": "^3.x",
+    "@nebula.js/cli-sense": "^3.x",
+    "@nebula.js/cli-serve": "^3.x"
   },
   "dependencies": {
     "@nebula.js/cli-create": "^3.0.0-alpha11",

--- a/commands/create/templates/sn/picasso/common/_package.json
+++ b/commands/create/templates/sn/picasso/common/_package.json
@@ -35,6 +35,6 @@
     "picasso-plugin-q": "1.2.1"
   },
   "peerDependencies": {
-    "@nebula.js/stardust": "^2.x"
+    "@nebula.js/stardust": "^3.x"
   }
 }

